src/05-derivatives/05-functional-derivatives.md

```diff
@@ -0,0 +1,390 @@
+following special labels correspond to common ROIs that may be defined by many
+atlases or segmentation algorithms:
+
+| ROI Value      | Description                                                                            |
+| -------------- | -------------------------------------------------------------------------------------- |
+| WhiteMatter    | Signal derived from white matter ROI.                                                  |
+| CSF            | Signal derived from cerebro-spinal fluid ROI.                                          |
+| Background     | Signal derived from background (out of brain) ROI.                                     |
+| GrayMatter     | Signal derived from gray matter ROI.                                                   |
+| Ventricles     | Signal derived from ventricles ROI.                                                    |
+| CircleOfWillis | Signal derived from circle of Willis ROI.                                              |
+| GlobalSignal   | Vector of mean values within the brain mask. Can be used for Global Signal Regression. |
+
+#### Column names
+
+Column names are unique alphanumeric values that are defined in a relevant JSON
+sidecar file. A naming convention might be to concatenate the atlas, ROI index,
+summarization method (defined below) and transformations (see [Time series
+transformations](#time-series-transformations)) using snake case. For example,
+`harvard_oxford_cortical_4_PC` could indicate ROI 4 of the Harvard-Oxford
+cortical atlas, summarized by taking the first principal component.
+
+#### Summarization methods
+
+To indicate the summarization method applied to construct a single time series
+for an ROI, the following column suffixes are defined:
+
+| Column suffix | Description of the transformation                                                                                                                       |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `_mean`       | The mean of voxel time series                                                                                                                           |
+| `_median`     | The median of voxel time series                                                                                                                         |
+| `_pc[_<x>]`   | The ith eigenvariate from principal component analysis, where `x` is 0 indexed. If `x` is not specified, the first component is implied (i.e., `pc_0`). |
+| `_spat_reg`   | Spatial regression                                                                                                                                      |
+
+Example:
+
+```Text
+sub-001/
+    func/
+        sub-001_task-rest_run-1_timeseries.tsv
+```
+
+```Text
+white_matter_mean global_signal_mean ventricles_mean
```

Comment:
Given that these columns correspond to established ROI names, should those ROI names be used instead? E.g., `white_matter_mean` --> `WhiteMatter_mean`.

I see above that snake case is preferred, so I guess it's a choice between name consistency or case consistency. Unless ROI names were changed from camel case to snake case by default?

Reply:
There was a decision around Derivatives RC1 to make all specified columns snake case instead of camel case, to be [consistent with BIDS Raw](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#tabular-files):

>  It is RECOMMENDED that the column names in the header of the TSV file are written in snake_case with the first letter in lower case (e.g., variable_name, not Variable_name)

I'm not sure if this is up for revisiting.

Reply:
Ah, I see. And metadata fields are established as camel case, so it's just a difference that will have to remain. That's fine then. Thanks for clarifying.